### DISCUSSION TITLE
apm and agent compatibility

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11806] [Q&A] [answered]
apache-skywalking-apm-9.3.0 compatible with agent 8.2.0,8.3.0,8.4.0,8.5.0,8.6.0,8.7.0,8.8.0,8.9.0,v9.0.0 versions?
After agent upgrade; do you need to re-create the es index?

Is this a compatibility statement between apm and agent?
<img width="1003" alt="image" src="https://github.com/apache/skywalking/assets/29425860/05eb4ee8-a235-4ea6-9892-1e55369b9faa">

**Comment by wu-sheng** [comment, marked as answer]
Yes. it is. But it is only correct usually from the release date of each version. Don't pick the agent released later than the OAP released. 
That part can't be covered by released OAPs.
